typedoc.json

```diff
@@ -1,8 +1,13 @@
 {
+  "inputFiles": ["src/index.ts"],
   "out": "docs",
-  "mode": "file",
+  "mode": "library",
+  "plugin": "typedoc-plugin-markdown",
   "readme": "none",
-  "theme": "markdown",
-  "mdEngine": "github",
-  "excludeNotExported": true
+  "gitRevision": "master",
+  "exclude": "src/**/!(secure|checkpointTrie|baseTrie).ts",
```

Comment:
I hope they do add a feature to automatically detect public API so we don't have to exclude by file name

Reply:
Yeah I agree, the `library` mode is a great first step so hopefully we'll see more in the future in that direction.